tools/commit_slider_tool/utils/modes.py

```diff
@@ -0,0 +1,146 @@
+import os
+from utils.helpers import fetchAppOutput, handleCommit, runCommandList
+from utils.helpers import getCommitLogger
+from utils.helpers import CashError
+from utils.helpers import CfgError
+import subprocess
+import re
+import json
+from utils.common_mode import Mode
+
+class CheckOutputMode(Mode):
```

Comment:
I thought that search can be implemented with [`git bisect`](https://git-scm.com/docs/git-bisect) command

Reply:
Why cannot you create a wrapper under this command?

Reply:
1. It's unnecessary. Git bisect implements binary search (5 lines on python). It's not a big deal for this feature.
2. git bisect cannot be used for all cases. For example, we're going to add traversal with showing off all differences during commit sequence. It extends simple bisect algorythm.